customize_image.sh

```diff
@@ -293,6 +293,7 @@ chroot "$IMAGEDIR" apt-mark hold raspi-copies-and-fills
 chroot "$IMAGEDIR" apt-get -y upgrade
 chroot "$IMAGEDIR" apt-mark unhold raspi-copies-and-fills
 chroot "$IMAGEDIR" apt-get clean
+chroot "$IMAGEDIR" apt clean
```

Comment:
Why don't we use just apt clean then?

Reply:
Thanks. It's now part of [301e669](https://github.com/RevolutionPi/imagebakery/pull/116/commits/301e6696c162d0db8356bc57d5db07d532484f76)